### PR TITLE
style: navbar redesign

### PR DESCRIPTION
**Pull request by franky47**
The one deviation from the design is keeping the light/dark mode control.

### CODE REVIEW

File: src/css/colors.scss [hunk 1]
```diff
@@ -6,5 +6,21 @@
   --ifm-color-primary-light: #9ea9ff;
   --ifm-color-primary-lighter: #b0baff;
   --ifm-color-primary-lightest: #e9ebff;
+  --ifm-navbar-background-color: #fff;
   --docusaurus-highlighted-code-line-bg: rgba(0, 0, 0, 0.1);
+  --docsearch-searchbox-background: #fff !important;
```
Comment (peterpeterparker) [inline src/css/colors.scss:11]: no other way than using `!important`?

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

File: src/css/colors.scss (hunk 1 above)
Comment (franky47) [reply src/css/colors.scss]: Fixed by moving the CSS var under the relevant class, to increase specificity and reuse the definition (same as navbar background).

Comment (peterpeterparker) [reply src/css/colors.scss]: Sounds good, thanks

**Pull request merged**